rescript/plugin_setup.py

```diff
@@ -386,26 +386,30 @@
     function=cull_seqs,
     inputs={
         'sequences': FeatureData[Sequence | RNASequence]
-        },
+    },
     parameters={
         'num_degenerates': Int % Range(1, None),
-        'homopolymer_length': Int % Range(2, None)
-        },
+        'homopolymer_length': Int % Range(2, None),
+        'n_jobs': Int % Range(1, None)
+    },
     outputs=[('clean_sequences', FeatureData[Sequence])],
     input_descriptions={
         'sequences': 'DNA or RNA Sequences to be screened for removal based '
                      'on degenerate base and homopolymer screening criteria.'
-        },
+    },
     parameter_descriptions={
         'num_degenerates': 'Sequences with N, or more, degenerate bases will '
                            'be removed.',
         'homopolymer_length': 'Sequences containing a homopolymer sequence of '
                               'length N, or greater, will be removed.',
+        'n_jobs': 'Number of concurrent processes to use while processing '
```

Comment:
Perhaps add something like this: "Output sequence order may change when using multiple jobs."

Reply:
Thanks @mikerobeson! - added and merging.